test/src/test/java/test/org/dst/benchmark/DSTBenchmark.java

```diff
@@ -0,0 +1,42 @@
+package test.org.dst.benchmark;
+
+import org.dst.client.DefaultDstClient;
+import org.dst.client.DstClient;
+import test.org.dst.benchmark.core.Benchmark;
+import test.org.dst.supplier.TestUtil;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+public class DSTBenchmark extends Benchmark {
```

Comment:
Why do we extend from Benchmark here?

Reply:
I don't understand what it is like here. Is it okay now?

Reply:
> Why do we extend from Benchmark here?
I don't understand what you want me to do here. Is it okay now?